src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/References/DesignTimeAssemblyResolutionTests.cs

```diff
@@ -195,7 +195,6 @@ public void ResolveAssemblyPathInTargetFx_NameThatMatches_ReturnsResolvedPaths(s
         [Theory]    // Input                                                                        // Name             // Version          // Path
         [InlineData("System",                                                                       "System.Core",      "",                 @"C:\System.Core.dll")]
         [InlineData("System",                                                                       "system",           "",                 @"C:\System.dll")]
-        [InlineData("system",                                                                       "System",           "",                 @"C:\System.dll")]
         [InlineData("encyclopædia",                                                                 "encyclopaedia",    "",                 @"C:\System.dll")]
```

Comment:
I assume this is for compat with previous behavior, which is prudent for initial commit, but shouldn't we match assembly names as the runtime does and not with a linguistic comparison? Should we track that?